app/views/shared/_modal.html.slim

```diff
@@ -0,0 +1,20 @@
+- root_attributes ||= {}
+- root_attributes[:id] ||= id
```

Comment:
If neither `root_attributes` nor `id` is passed, this would fail, wouldn't it?

Reply:
Yes. I will just remove `root_attributes[:id] ||= id` because it is not really needed anyway.